corehq/apps/app_manager/views/releases.py

```diff
@@ -124,14 +125,28 @@ def _get_batch(start_build=None, skip=None):
             app_es = app_es.is_released()
         if build_comment:
             app_es = app_es.build_comment(build_comment)
+
         results = app_es.exclude_source().run()
+        total_apps = results.total
         app_ids = results.doc_ids
         apps = get_docs(Application.get_db(), app_ids)
+
+        if build_comment:
+            try:
+                version = int(build_comment)
+                build_doc = get_build_doc_by_version(domain, app_id, version)
+                if build_doc and build_doc['_id'] not in app_ids:
+                    total_apps = total_apps + 1
+                    apps = apps + [build_doc]
+                apps = sorted(apps, key=lambda a: -a['version'])
+                apps = apps[:limit]
+            except ValueError:
+                pass
```

Comment:
nit: this might be one of those rare situations where `try`...`else` actually makes sense assuming you only intend to catch the `ValueError` in the first line.

Reply:
good point, a85be0e9221